rdm/main.py

```diff
@@ -77,6 +80,11 @@ def parse_arguments(arguments):
     pull_parser = subparsers.add_parser('pull', help=pull_help)
     pull_parser.add_argument('config', help='Path to project `config.yml` file.')
 
+    gap_help = 'collect documentation snippets into a yaml file'
```

Comment:
probably a leftover text from copy/paste

Reply:
I will improve that.